flink-connectors/flink-sql-connector-hive-1.2.2/src/main/resources/META-INF/licenses/LICENSE.jodd-core

```diff
@@ -0,0 +1,31 @@
+Copyright (c) 2003-2014, Jodd Team
```

Comment:
LICENSE.jodd is enough?

Reply:
What's the naming convention of these license files? I also added `LICENSE.protobuf-java` which perhaps should be changed to `LICENSE.protobuf`?

Reply:
The only important thing is that the license name makes it clear to which dependencies it applies to. generally I think we tried to be as short as possible.